spring-cloud-starter-dataflow-server/pom.xml

```diff
@@ -19,12 +19,12 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
-			<artifactId>spring-cloud-deployer-local</artifactId>
+			<artifactId>spring-cloud-dataflow-server-core</artifactId>
```

Comment:
Please revert - the file is the same as before other than "formatting" which equates to another file to review and resonate with.